pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/source/SierraQueryOps.scala

```diff
@@ -12,6 +14,16 @@ trait SierraQueryOps {
     def varfieldsWithTag(tag: String): List[VarField] =
       varfieldsWithTags(tag)
 
+    def nonrepeatableVarfieldWithTag(tag: String): Option[VarField] =
+      varfieldsWithTag(tag) match {
+        case Seq(vf) => Some(vf)
+        case Nil     => None
+        case fields =>
```

Comment:
Does Sierra allow this to happen?

Reply:
I don't know, which is why I'm checking it here.